### PR TITLE
Remove scripts under RelVal/testbed

### PR DESCRIPTION
**Pull request by amaltaro**
As per @ AndrewLevin  request, removing scripts under RelVal/testbed.

### CODE REVIEW

**Comment by julianbadillo** [comment]
Ok, let me backup mine into the / folder and I'll merge this

**Comment by julianbadillo** [comment]
@amaltaro what about resubmitTaskChain? I was using it to clone tc's but it looks like the regular resubmit.py works ok.

**Comment by amaltaro** [comment]
I guess the only difference is the loop in line 75 to fix old arguments.

**Comment by julianbadillo** [comment]
This one?
https://github.com/CMSCompOps/WmAgentScripts/blob/master/resubmit.py#L75

File: RelVal/testbed/resubmitTaskChain.py [hunk 1]
```diff
@@ -1,199 +0,0 @@
...
-        elif value != None:
-            schema[key] = value
-#    print "Retrieved schema:"
-#    pprint.pprint(schema)
-#    sys.exit(1)
-    request = deepcopy(schema)
-
-    ### Now changing the parameters according to HG1309
-    x = 1
-    while x <= schema['TaskChain']:
```
Comment (amaltaro) [inline RelVal/testbed/resubmitTaskChain.py]: Nope, this one! Which is not really important, but helpful when we need to clone an old workflow with old argument names.

**Comment by julianbadillo** [comment]
Then I can copy those options to the resubmit.py, so we can use only one script to clone them all

**Comment by amaltaro** [comment]
Better putting a check for RequestType before, otherwise it fails :-)

**Pull request merged**